binr/ragg2/ragg2.c

```diff
@@ -40,6 +40,7 @@ static int usage(int v) {
 			" -q [fragment]   debruijn pattern offset\n"
 			" -r              show raw bytes instead of hexpairs\n"
 			" -s              show assembler\n"
+            		" -S              append a string\n"
```

Comment:
That help is wrong. Doesnt implies the flag have any arg

Reply:
Okay, I'll add the [string] next to it.